generators/app/templates/gulpfile.js

```diff
@@ -80,6 +80,10 @@ gulp.task('lint', function (done) {
     .pipe(eslint.failOnError());
 });
 
+gulp.task('test', ['lint'], function () {
+  gutil.log(gutil.colors.red('Nothing to test...'));
```

Comment:
That's the plan, just don't have anything ready yet, so not in this PR. I just wanted the script execution point to run the linter for now.